host-bmc/host_pdr_handler.hpp

```diff
@@ -199,6 +199,10 @@ class HostPDRHandler
     /** @brief map that captures various terminus information **/
     TLPDRMap tlPDRInfo;
 
+    /** @brief Delete DBUS objects
+     */
```

Comment:
Please add the input parameter description

Reply:
Updated